javascript/javascript3/week2/RewriteTime.js

```diff
@@ -0,0 +1,43 @@
+
+ 
+
+  ///////////////////////////////////////////////////////////
+
+function getCurrentLocation(){
+    try{
+        return new Promise(function(resolve,reject){
+              function success(pos) {
+                var crd = pos.coords;
+                console.log('Your current position is:');
+                console.log(`Latitude : ${crd.latitude}`);
+                console.log(`Longitude: ${crd.longitude}`);
+                console.log(`More or less ${crd.accuracy} meters.`);
+              }
+              function error(err) {
+                console.warn(`ERROR(${err.code}): ${err.message}`);
+              }
+              navigator.geolocation.getCurrentPosition(success, error);
+            resolve('location allocated!');
+            reject('location not allocated!');
+        });
```

Comment:
In this case the `reject` function would be called togeher with `resolve`
Reject should be called when geolocation data not avaialable. For example
In this case better put the `resolve` in `success` callback and `reject` in `error` callback